quarkchain/cluster/tests/test_cluster.py

```diff
@@ -2457,85 +2457,26 @@ def test_grpc_slave(self):
         acc1 = Address.create_from_identity(id1, full_shard_key=0)
 
         with ClusterContext(1, acc1, connect_grpc=True) as clusters:
-            block_header_list = [clusters[0].get_shard_state(2 | 0).header_tip]
-            shard_state0 = clusters[0].get_shard_state(0b10)
-            for i in range(7):
-                b1 = _tip_gen(shard_state0)
-                add_result = call_async(
-                    clusters[0].master.add_raw_minor_block(
-                        b1.header.branch, b1.serialize()
-                    )
-                )
-                self.assertTrue(add_result)
-                block_header_list.append(b1.header)
-
-            block_header_list.append(clusters[0].get_shard_state(2 | 1).header_tip)
-            shard_state0 = clusters[0].get_shard_state(0b11)
-            b2 = _tip_gen(shard_state0)
-            add_result = call_async(
-                clusters[0].master.add_raw_minor_block(b2.header.branch, b2.serialize())
-            )
-            self.assertTrue(add_result)
-            block_header_list.append(b2.header)
-
-            root_block = clusters[0].master.root_state.create_block_to_mine(
-                block_header_list, acc1
-            )
             # Test Case 1 ###################################################
             # This case tests the correct connection
-            normalserver1 = NormalServer()
-            server1, port_number1 = self.build_test_server(
+            root_block = clusters[0].master.root_state.create_block_to_mine([])
+            normalserver1 = MockGrpcServer()
```

Comment:
nit: rename `normalserver1` to `server`, `server1` / `server2` to `grpc_server1` / `grpc_server2`